apisix/discovery/kubernetes/init.lua

```diff
@@ -331,9 +332,18 @@ local function start_fetch(handle)
     ngx.timer.at(0, timer_runner)
 end
 
+local function get_endpoint_dict(shm)
```

Comment:
We can pass the id as the argument, and put the constant `kubernetes` in this function? There is no need to repeat it.